aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md

```diff
@@ -1,4 +1,4 @@
-# Azure OAuth 2.0 Sample for Azure AD Spring Boot Starter Resource Server Obo client library for Java
+# Azure OAuth 2.0 Sample for Azure AD Spring Boot Starter Resource Server Obo
```

Comment:
Do you think `On-Behalf-Of flow` or `OBO` is better? 

https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow

Reply:
There are much more improvements in all titles.  In this PR, I just **Delete "client library for java" suffix in all readme.md** as described in the title.